File: code_challenging(JAVA)/palindromo/src/MenorNumero.java
import java.util.Scanner;

public class MenorNumero {

	public static void main(String[] args) {
		int[] vect = {10, 5, 22, 1, 9};
		int menor = vect[0];
		for( int n : vect ) {
			menor = n < menor ? n : menor;
		}
		System.out.println(menor);
	}

}

File: code_challengin(JS)/menorNumero.js
function menorNumero(numeros){
    /*let menor = numeros[0]
    numeros.forEach((n) => {
        menor = n < menor ? n : menor
    })*/
    
    let menor = numeros.sort((a, b) => a - b)
    return menor[0]
}

console.log(menorNumero([10, 5, 22, 1, 9]))